elections/tools/README.md

```diff
@@ -0,0 +1,37 @@
+
+# Election Tools
+
+This directory contains the tools required to generate the kata electorate
+
+## generate_electorate.py
+
+This tool need the following python libraries pytz github3.py pyyaml.  To
+install them in a virtutal envoronment do something like:
+
+```
+python3 -m venv .venv
+.venv/bin/pip install pytz github3.py pyyaml
+```
+
+Before running the tool you will need to create a
+[github API token](https://github.blog/2013-05-16-personal-api-tokens/)
+
+replace ``__API_TOKEN__`` with your personal token
+
+Also update the election start and end times.  Then run the tool with:
+``.venv/bin/python ./generate_electorate.py``
+
+The code looks a all commits in all kata-containers repos *except*
+kata-containers/linux and kata-containers/qemu.  As both of these are forks (in
+the GitHUb sense) they'll have lots of contributors that may not be kata
+contributors.
+
+For contributors that have more than one email address I pick one as default
+but supply all the others so you can be smarter about emailing them it you
+want.
+
+The sources for email addresses are:
+- github account
+- git commit data
```

Comment:
Nit: "Git" as this is referring to the name of the tool, not the command itself.